projects/opal-frontend-common/components/govuk/govuk-list/govuk-list-link/govuk-list-link.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input, Output, EventEmitter, ChangeDetectionStrategy } from '@angular/core';
+
+@Component({
+  selector: 'opal-lib-govuk-list-link',
+  imports: [],
+  templateUrl: './govuk-list-link.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class GovukListLinkComponent {
+  @Input({ required: true }) linkText!: string;
+  @Output() linkClickEvent = new EventEmitter<void>();
+
+  handleClick(event: MouseEvent): void {
+    event.preventDefault();
+    this.linkClickEvent.emit();
+  }
```

Comment:
Missing public from this method, but also, we can't rely on MouseEvents entirely. It needs to be all DOM events. So it would be better to refactor this to: 

```suggestion
  public handleClick(event: Event): void {
    if (event.preventDefault) {
       event.preventDefault();
    }
    this.linkClickEvent.emit();
  }
```